projects/kit/components/checkbox/checkbox-options.ts

```diff
@@ -1,5 +1,19 @@
+import {InjectionToken} from '@angular/core';
 import {TuiSizeL} from '@taiga-ui/core';
 
 export interface CheckboxOptions {
-    size?: TuiSizeL;
+    readonly size: TuiSizeL;
```

Comment:
I wonder why doesn't this go into conflict with size being non-readonly in checkbox component 🤔

Reply:
I was wondering the exact same thing TBO. I'd find it more intuitive to remove the readonly modifier from the field as I don't think it adds much value.

Reply:
Let's remove `readonly` here and add utility type `Readonly<CheckboOptions>` to the token then.